app/javascript/components/Results/Results.tsx

```diff
@@ -1,27 +1,31 @@
+/* eslint-disable complexity */
```

Comment:
Due to the significant amount of complexity introduced by Bing pagination being different than pagination for all other types of searches, I've disabled the `complexity` rule for this file. And this may end up being removed as we develop the new SERP further.